pkgs/os-specific/linux/power-profiles-daemon/default.nix

```diff
@@ -95,13 +91,13 @@ stdenv.mkDerivation rec {
 
   PKG_CONFIG_POLKIT_GOBJECT_1_POLICYDIR = "${placeholder "out"}/share/polkit-1/actions";
 
-  # Avoid double wrapping
-  dontWrapGApps = true;
-
   postPatch = ''
     patchShebangs --build \
       tests/integration-test.py \
       tests/unittest_inspector.py
+
+    patchShebangs --host \
+      src/powerprofilesctl
   '';
 
   postCheck = ''
```

Comment:
This can probably be dropped too.

Reply:
We can't, the integration test suite is running powerprofilectl (in the checkPhase). We'll lack pygobject in the python path if we don't patch the shebang in the patchPhase.

Reply:
https://gitlab.freedesktop.org/upower/power-profiles-daemon/-/commit/718fde54d22e1bf486cd56463866a70cd1d18429

Reply:
This comment referred to the `postCheck` block.

---

But if `powerprofilectl` is run in `checkPhase` as well then `patchShebangs --host` will not work there. But maybe we do not need to care about that as IIRC tests do not run anyway in cross.

Reply:
Duh! Sorry about that.

You're absolutely right.